pkg/generate/code/check.go

```diff
@@ -65,21 +66,21 @@ func CheckExceptionMessage(
 // return r.ko.Spec.APIID == nil || r.ko.Status.RouteID == nil
 func CheckRequiredFieldsMissingFromShape(
 	r *model.CRD,
-	opType model.OpType,
+	opType operations.OpType,
 	koVarName string,
 	indentLevel int,
 ) string {
 	var op *awssdkmodel.Operation
 	switch opType {
-	case model.OpTypeGet:
+	case operations.OpTypeGet:
```

Comment:
package `operations` should be renamed to `operation` and optionally we can rename `OpTypeGet` to `TypeGet` - https://rakyll.org/style-packages/